src/unix/bsd/mod.rs

```diff
@@ -324,18 +324,30 @@ f! {
         }
     }
 
-    pub fn WIFEXITED(status: ::c_int) -> bool {
-        (status & 0x7f) == 0
+    pub fn WIFSTOPPED(status: ::c_int) -> bool {
+        (status & 0o177) == 0o177
```

Comment:
Locally on OSX for me it looks like this is

```c
(_WSTATUS(x) == _WSTOPPED && WSTOPSIG(x) != 0x13)
```

so maybe this should also check `WSTOPSIG`?